eclass/qt5-build.eclass

```diff
@@ -689,6 +694,33 @@ qt5_base_configure() {
 
 }
 
+# @FUNCTION: qt5_qmake_args
+# @INTERNAL
+# @DESCRIPTION:
+# Helper function to get the various toolchain-related variables.
+qt5_qmake_args() {
```

Comment:
Ideally we'd call this from ``qt5_qmake`` too but I can't get the quoting to work correctly.